src/roll-chromium.ts

```diff
@@ -156,34 +148,69 @@ export async function rollChromium(
 }
 
 export async function rollChromium4(
-  electronBranch: string,
+  electronBranch: {name: string, commit: {sha: string}},
   chromiumVersion: string,
-): Promise<string | null> {
-  d(`triggered for electronBranch=${electronBranch} chromiumVersion=${chromiumVersion}`);
+): Promise<void> {
+  d(`roll triggered triggered for electronBranch=${electronBranch.name} chromiumVersion=${chromiumVersion}`);
   const github = await getOctokit();
-  // Get current SHA of {electronBranch} on electron/electron
-  const electronReference = await github.git.getRef({
+
+  // Look for a pre-existing PR that targets this branch to see if we can update that.
+  const existingPrsForBranch = await github.pulls.list({
```

Comment:
Can we add a `//TODO` to paginate here

Reply:
I wish there were a way to filter only electron-bot's PRs but the API doesn't allow it :(